### DISCUSSION TITLE
Add "device with incorrect configuration" ?

### DISCUSSION THREAD

**Discussion by Chris0706** [Zefau/ioBroker.jarvis#1200] [Q&A]
Hi Zefau,

I am using jarvis in combination with a RaspberryMatic and I recently added a room thermostat. This shows as a device with incorrect configuration. 

![image](https://user-images.githubusercontent.com/9281382/143046339-cb2d7c2b-3eed-428f-91dc-43de684fd663.png)

Is it possible to write a custom configuration? Or can I deliver some data so you could work on it?

Best regards
Christian

**Comment by Zefau** [comment]
Please provide the states you would like to import and possibly screenshots, see https://github.com/Zefau/ioBroker.jarvis/issues/1196 as an example.

**Comment by Chris0706** [comment]
Does this help? I guess most of the states should be pretty similar to normal thermostat?
![Wandthermostat_States](https://user-images.githubusercontent.com/9281382/145545863-5edb6f8f-bcc5-4ca3-a6f7-f20ad1851419.png)
